pyproject.toml

```diff
@@ -130,7 +130,7 @@ legacy_tox_ini = """
     [testenv:black]
     deps =
         black
-    commands = black {posargs:.}
+    commands = black --version && black {posargs:.}
```

Comment:
@john-westcott-iv This disabled black. This is parsed as `black --version '&&' black .` which ends up the same as `black --version` and all other arguments get dropped.